src/executors/default-with-mysql.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
```

Comment:
I suggest to add three more parameters to this executor for the environment variables DB_TEST_URL, MYSQL_ROOT_PASSWORD, MYSQL_DATABASE and we can add their current values (respectively root:password@/pac, password, pac) as default values of those parameters.

Reply:
Done